src/main/java/blackjack/domain/card/Cards.java

```diff
@@ -0,0 +1,40 @@
+package blackjack.domain.card;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cards {
+
+    private static final int INIT_CARDS_SIZE = 2;
+
+    private final List<Card> value;
+
+    public Cards(List<Card> cards) {
+        this.value = new ArrayList<>(cards);
+        validateSize();
+    }
+
+    private void validateSize() {
+        if (value.size() != INIT_CARDS_SIZE) {
+            throw new IllegalArgumentException("카드의 개수는 2장이어야 합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("카드의 개수는 " + INIT_CARDS_SIZE + "장이어야 합니다.");
```
게임 규칙이 바꼈을 때 메시지의 카드 갯수를 놓치지 않도록 상수를 사용하시기 바랍니다.

Reply:
예외 메시지도 상수를 활용하여 유동적으로 변경될 수 있도록 개선하였습니다!

```java
public class Cards {

    private static final int INIT_CARDS_SIZE = 2;
    ....
    private void validateSize() {
        if (value.size() != INIT_CARDS_SIZE) {
            throw new IllegalArgumentException("카드의 개수는 " + INIT_CARDS_SIZE + "장이어야 합니다.");
        }
    }
    ....
}
```